spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/LSTMPeephole.scala

```diff
@@ -51,10 +59,12 @@ class LSTMPeephole[T : ClassTag] (
 
     val i2g = Sequential()
       .add(Dropout(p))
-      .add(Linear(inputSize, hiddenSize))
+      .add(Linear(inputSize, hiddenSize, wRegularizer = wRegularizer,
```

Comment:
Fixed regularization Configuration for LSTM, wRegularizer for input weight, uRegularizer for hidden weight, bRegularizer for bias.